spec/xclarity_client_base_spec.rb

```diff
@@ -28,13 +28,23 @@
         end
       end
 
-      context 'with response 302' do
```

Comment:
We don't need test response 302 too?

Reply:
Hey Andrey, thanks for your review. I modify this test to catch exceptions of type 'ConnectionFailedUnknown', which is raised when the status is different of 200 or for unknown error. Example: status 404 or 302 and son on. As an example I prefer 404.